src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -943,7 +943,7 @@ bool InstanceKlass::link_class_impl(TRAPS) {
   {
     HandleMark hm(THREAD);
     Handle h_init_lock(THREAD, init_lock());
-    ObjectLocker ol(h_init_lock, THREAD);
+    ObjectLocker ol(h_init_lock, jt);
```

Comment:
I was going to mumble about adding a new 'jt' here, but this isn't an
ObjectMonitor related file so you probably held off here.